phishing-detection/phishing_detection/train.py

```diff
@@ -21,11 +30,36 @@ def train(model: Model, X_train: np.array, y_train: np.array, X_val: np.array, y
     model.compile(loss=params['loss_function'], optimizer=params['optimizer'], metrics=['accuracy'])
 
 
-    hist = model.fit(X_train, y_train,
```

Comment:
why are we saying model here? This works differently from pytorch. Keras models return a history.

Reply:
From my understanding history is just an object that records logs and not the model itself. We want to save the trained model to use it for testing

Reply:
yea, but renaming it doesn't give the model. model.fit() returns the history and updates the model in-place without returning the updated model. That's what I remember from the docs. This is not handy, but Keras works that way

Reply:
We are not using the return value of model.fit, therefore the local variable hist was removed.